dubbo-dependencies-bom/pom.xml

```diff
@@ -609,6 +609,13 @@
                 <artifactId>nacos-client</artifactId>
                 <version>${nacos_version}</version>
             </dependency>
+            <!--for dubbo-configcenter-apollo-->
+            <dependency>
+                <groupId>com.ctrip.framework.apollo</groupId>
+                <artifactId>apollo-mockserver</artifactId>
+                <version>${apollo_client_version}</version>
+                <scope>test</scope>
```

Comment:
pls. limit the test scope library to the apollo module, since this bom will be distributed to all dubbo users.